hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -4429,4 +4429,13 @@
       maximum number of buckets across all volumes.
     </description>
   </property>
+  <property>
+    <name>ozone.om.requests.ignore.terminate.on.failure</name>
+    <value>DeleteOpenKeys,EchoRPC</value>
```

Comment:
DeleteOpenKeys alters the openKeyTable cache inside the request. I feel any request that alters the state of OM (say cache/DB) should not come under this category as ignoring such failures might cause inconsistency in OM metadata